hail/python/hail/backend/py4j_backend.py

```diff
@@ -293,12 +282,6 @@ def _register_ir_function(self, name, type_parameters, argument_names, argument_
             code,
         )
 
-    def _parse_table_ir(self, code):
-        return self._jbackend.parse_table_ir(code)
-
-    def _parse_matrix_ir(self, code):
-        return self._jbackend.parse_matrix_ir(code)
-
     def _parse_blockmatrix_ir(self, code):
```

Comment:
this is still used to compute blockmatrix sparsity. All others were just used in tests to test that irs parse (which is done by the `_type` methods)
I wonder if we can move sparsity into the block matrix type and then this too can be removed?